test/e2e/service.go

```diff
@@ -1283,7 +1288,11 @@ metadata:
 		//	nexthop via 172.19.0.4 dev eth0 weight 1
 		//	nexthop via 172.19.0.2 dev eth0 weight 1
 		cmd := []string{containerRuntime, "exec", routerContainer}
-		bgpRouteCommand := strings.Split("ip route show 192.168.10.0", " ")
+		ipVer := ""
+		if utilnet.IsIPv6String(svcLoadBalancerIP) {
+			ipVer = " -6"
+		}
+		bgpRouteCommand := strings.Split(fmt.Sprintf("ip%s route show %s", ipVer, svcLoadBalancerIP), " ")
```

Comment:
%s is glued to ip because its treated as parameter and this isn't expected for ip commands. (because of the split here)